ert_shared/storage/model.py

```diff
@@ -252,3 +252,17 @@ class ErtBlob(Blobs):
 
     def __repr__(self):
         return "<Value(id='{}', data='{}')>".format(self.id, self.data)
+
+
+
+prior_ensemble_association_table = Table('prior_ensemble_association_table', Entities.metadata,
+    Column('prior_id', String, ForeignKey('parameter_priors.key')),
+    Column('ensemble_id', Integer, ForeignKey('ensembles.id'))
+)
+
+class ParameterPrior(Entities):
+    __tablename__ = "parameter_priors"
+
+    key = Column('key', String, primary_key=True)
```

Comment:
Should there not be an association between the ParameterDefinition that is sampled from a Prior other than the implicit key link?

Reply:
There is another implicit association also, through the ensemble it belongs to. The root ensemble will have the prior explicitly. Not sure if that is enough either though. What do you think? Perhaps we should just get it in as it is now, and create more issues to discuss how it should be?